Resources/public/js/alloyeditor/plugins/floatingtoolbar.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+/* global CKEDITOR */
+YUI.add('ez-alloyeditor-plugin-floatingtoolbar', function (Y) {
+    "use strict";
+
+    if (CKEDITOR.plugins.get('ezfloatingtoolbar')) {
+        return;
+    }
+
+    var FLOATING_TOOLBAR_SELECTOR = '.ae-toolbar-floating',
+        FLOATING_TOOLBAR_FIXED_CLASS = 'ae-toolbar-floating-fixed';
+
+    function findScrollParent(editor) {
+        var container = (new Y.Node(editor)).ancestor('.ez-main-content');
+        if (container.getStyle('overflow') === 'auto') {
+            return container.getDOMNode();
+        }
+
+        return window;
+    }
+
+    function findFocusedEditor() {
+        for (var name in CKEDITOR.instances) {
+            if (CKEDITOR.instances[name].focusManager.hasFocus) {
+                return CKEDITOR.instances[name];
+            }
+        }
+
+        return null;
+    }
+
+    function scrollHandler () {
+        var toolbar = document.querySelector(FLOATING_TOOLBAR_SELECTOR);
```

Comment:
Please, add an empty space before `if` statement. Please, treat it as a general rule to follow in your PR.